sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2355,9 +2371,11 @@ private void SetNameFieldsIfNull()
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+#pragma warning disable AZC0002 // Client method should have an optional CancellationToken (both name and it being optional matters) or a RequestContext as the last parameter.
```

Comment:
Would it be possible to make this cancellationToken optional, or would that be a breaking change?

Why must the cancellationToken be optional if there is an overload that doesn't take one?